src/main/java/com/kodilla/ecommercee/domain/dto/ProductsInCartDto.java

```diff
@@ -9,4 +9,5 @@ public class ProductsInCartDto {
 
     private Long cartId;
     private Long productId;
+    private Long quantity;
```

Comment:
As above - should be `int productQuantity`.

Reply:
Resolved